ocs_ci/framework/conf/default_config.yaml

```diff
@@ -103,3 +103,7 @@ ENV_DATA:
   # measurement_dir: '/tmp/ocs_ci_monitoring_measurement/'
   # Default RHCOS image to be used for VmWare deployment
   vm_template: 'rhcos-4.3.0-x86_64-vmware'
+
+# This section is related to upgrade
+UPGRADE:
+  upgrade_to_latest: true
```

Comment:
won't this affect any not upgrade related scenarios?

Reply:
no it won't